docs/guide/installation/advanced.rst

```diff
@@ -36,14 +36,18 @@ It is however *strongly* recommended to have all the dependencies installed (wit
 
 There are multiple options depending on how many optional dependencies you want to install:
 
-To install sunpy with ``pip`` including optional dependencies (recommended), simply run::
+To install sunpy with ``pip`` including all optional dependencies (*recommended*), simply run::
 
     pip install sunpy[all]
 
 To install sunpy with no optional dependencies::
 
     pip install sunpy
 
+To install sunpy with map- and timeseries-based dependencies::
+
+    pip install sunpy[map,timeseries]
+
 To install sunpy with net-based dependencies::
 
     pip install sunpy[net]
```

Comment:
I can't comment lower than this line, but I would remove `pip install sunpy[dev]` (line 63 below), and also remove the `[docs]` option, leaving this page as a user facing page.

There's also a lot of optionals enumerated in the text now, might be worth condensing them down to one example with a list of the options available?

Reply:
Have a look, see if you like it.